libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/usePanelTabs.tsx

```diff
@@ -57,11 +57,11 @@ export const useCreateWorkflowPanelTabs = ({ onCreateClick }: { onCreateClick: (
     }
     if (!isConsumption && selectedTabId && selectedTabId !== Constants.TEMPLATE_PANEL_TAB_NAMES.BASIC) {
       if (!existingWorkflowName) {
-        dispatch(validateWorkflowName(existingWorkflowNames ?? []));
+        dispatch(validateWorkflowName({ id: id as string, existingNames: existingWorkflowNames ?? [] }));
```

Comment:
We need to fix the code where we keep initializing the tabs when there is no template selected. It is because of this I had to add {} when there is no workflow.. I will fix that in a separate bug fix.. so we do not have to come to a place to doing ?? '' because this code should never hit when there is no workflow selected.
if this fails it is a bug in our code and we should fix it. There is no harm in assuming id will be a string here.